From 5ebc7e9d83fa5d5757c4a650fe53146d906a1136 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Mon, 21 Aug 2023 10:02:20 -0500
Subject: [PATCH] bbb.io: make sure all env follow ti/<dtb>

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 board/ti/j721e/j721e.env | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/ti/j721e/j721e.env b/board/ti/j721e/j721e.env
index 2333c57e8b..9b9a3cc62f 100644
--- a/board/ti/j721e/j721e.env
+++ b/board/ti/j721e/j721e.env
@@ -12,7 +12,7 @@ defined(CONFIG_TARGET_J721E_R5_EVM)
 #include <environment/ti/k3_rproc.env>
 #endif
 
-default_device_tree=k3-j721e-beagleboneai64.dtb
+default_device_tree=ti/k3-j721e-beagleboneai64.dtb
 findfdt=
 	setenv name_fdt ${default_device_tree};
 	if test $board_name = j721e; then
-- 
2.34.1

