From 22cd7885fd089fc51e1335c50aa6dea22e595ff8 Mon Sep 17 00:00:00 2001
From: "Theodore A. Roth" <troth@openavr.org>
Date: Mon, 11 Sep 2023 05:40:15 -0600
Subject: [PATCH] board: Change reported board name to BBONEAI-64

Signed-off-by: Theodore A. Roth <troth@openavr.org>
---
 board/ti/j721e/evm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/ti/j721e/evm.c b/board/ti/j721e/evm.c
index 5c6fd588c3..7504e24be6 100644
--- a/board/ti/j721e/evm.c
+++ b/board/ti/j721e/evm.c
@@ -197,7 +197,7 @@ int checkboard(void)
 
 	if (do_board_detect())
 		/* EEPROM not populated */
-		printf("Board: %s rev %s\n", "J721EX-PM1-SOM", "E2");
+		printf("Board: %s rev %s\n", "BBONEAI-64", "B0");
 	else
 		printf("Board: %s rev %s\n", ep->name, ep->version);
 
-- 
2.34.1

